projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/class-verbum-comments.php

```diff
@@ -64,6 +64,8 @@ public function __construct() {
 
 		// After the comment is saved, we add meta data to the comment.
 		add_action( 'comment_post', array( $this, 'add_verbum_meta_data' ) );
+		add_filter( 'pre_comment_content', array( $this, 'remove_blocks' ) );
+		add_filter( 'comment_text', array( $this, 'render_verbum_blocks' ) );
```

Comment:
How are we ensuring this will only apply to Verbum comments? I know we exclude P2 and forums, but this filter will run in other places as well.

Reply:
I don't mean this way, I mean this filter will run in unexpected places outside our scope. Check this example below, the Tumblr tipping field is a comment filed on wp.com simple site, this filter may run against it even though we never test that.

p1698351357360729-slack-C04NARZL78D